src/main/java/carpetaddonsnotfound/CarpetAddonsNotFoundSettings.java

```diff
@@ -75,7 +74,7 @@ public class CarpetAddonsNotFoundSettings {
   @Rule(categories = { FEATURE, SURVIVAL, CARPET_ADDONS_NOT_FOUND })
   public static ReplaceableFlowersOptions replaceableFlowers = ReplaceableFlowersOptions.FALSE;
 
-  @Rule(categories = { FEATURE, CREATIVE, CARPET_ADDONS_NOT_FOUND })
+  @Rule(categories = { SPECTATOR, CARPET_ADDONS_NOT_FOUND })
```

Comment:
The categories changed here are for the `spawnEggsSpawnMobsWithNoAI`, please revert this and do it for `spectatorPlayersUsePortals` just below it.